examples/validate_relative_binding.py

```diff
@@ -93,12 +94,15 @@ def fn(*args, **kwargs):
 
 
 if __name__ == "__main__":
+    default_output_path = f"rabfe_{datetime.datetime.now().strftime('%Y_%m_%d_%H_%M_%S')}.sdf"
```

Comment:
This is the convention here: https://github.com/proteneer/timemachine/blob/master/examples/hif2a/fit_to_multiple_rbfes.py#L188

Don't have a preference as this really is just to avoid rerunning the script and overwriting an existing file